components/tokens/index.js

```diff
@@ -78,7 +78,16 @@ const getVotingPowerAtLevel = async (req, response) => {
       userAddress
     );
 
-    response.json({ votingWeight });
+    const votingXTZWeight = await getUserTotalVotingPowerAtReferenceBlock(
```

Comment:
In the request we should have whether we want voting power for Xtz or Token. Don't think we should make multiple calls if not needed to avoid spamming the API and reducing load time.

Reply:
We are not making multiple/useless calls. The function calls a specific API depending on whether the poll.isXTZ is set to true or false

Reply:
Besides, right now, this info is been useful for the frontend and I believe returning the info like this, can be benefitial for current and future tweaks/features